setup/logging.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Kubernetes Logs
+description: Learn how to set up logging in Kubernetes
+---
+
+Kubernetes logs can help you understand what's happening under the hood of your Coder
+deployment. These logs can be utilized for debugging Coder services and
+monitoring the overall health of your cluster. Below are the logging
+configuration options available in the [Helm chart](https://github.com/cdr/enterprise-helm).
+
+> Coder supports writing logs to multiple output targets. Set these
+values to an empty string to disable.
+
+## Human logging
+
+This is the default value set in the Helm chart, as reflected below:
+
+```yaml
+logging:
+    human: /dev/stderr
+```
+
+When set, logs will be sent to the `/dev/stderr` file path, and formatted for human
+readability.
```

Comment:
This may be a noob question, but how would someone fetch these with `kubectl`? Might help to include this here.

Reply:
Agreed. This I'm not sure of either. Would like some clarification here.

Reply:
`/dev/stdout` & `/dev/stderr` both write to the pods stdout/stderr, accessible via `kubectl logs <podname>`